pdfmaker/css/holt/pdf.css

```diff
@@ -1,9 +1,1175 @@
+  margin-top: 18pt; }
+
+*[class*="AdCardListofTitlesacl"] {
+  text-align: center;
+  text-indent: 0;
+  font-style: italic;
+  line-height: 16pt;
+  margin-top: 8pt; }
 
 section[data-type="titlepage"] *[class*="TitlepageLogologo"] {
   background-image: url("https://raw.githubusercontent.com/macmillanpublishers/bookmaker_assets/master/pdfmaker/images/holt/logo.jpg");
   background-size: auto 0.5in;
   height: 0.55in;
+  background-repeat: no-repeat;
   background-position: center center;
-  margin-bottom: auto;
-}
\ No newline at end of file
+  margin-bottom: auto; }
+
+section[data-type="titlepage"] > h1 {
+  padding-left: 39pt;
+  padding-right: 39pt;
+  font-weight: normal;
+  font-size: 26pt;
+  hyphens: none; 
+  line-height: 32pt;
+  text-align: center;
+  margin-bottom: 0;
+  string-set: title content(); }
+
+section[data-type="titlepage"] *[class*="Titlepage"] {
+  text-indent: 0;}
+
+section[data-type="titlepage"] > h1:first-child {
+  margin-top: 70pt; }                 
+
+*[class*="TitlepageBookSubtitlestit"] + h1,
+*[class*="TitlepageSeriesTitleser"] + h1 {
+  margin-top: 20.58pt; }
+
+h1 + *[class*="TitlepageBookSubtitlestit"],
+h1 + *[class*="TitlepageSeriesTitleser"] {
+  margin-top: 19pt;
+  margin-bottom: 0; }
+
+*[class*="TitlepageBookSubtitlestit"],
+*[class*="TitlepageSeriesTitleser"] {
+  margin-top: 19pt;
+  text-indent: 0;
+  padding-left: 72.5pt;
+  padding-right: 72.5pt;
+  font-weight: normal;
+  font-size: 14pt;
+  line-height: 19pt;
+  text-align: center;   
+  hyphens: none; }
+
+*[class*="TitlepageBookSubtitlestit"]:first-child,
+*[class*="TitlepageSeriesTitleser"]:first-child {
+  margin-top: 4.75pt; }
+
+*[class="TitlepageBookTitletit"] + *[class*="TitlepageBookSubtitlestit"], 
+*[class="TitlepageBookTitletit"] + *[class*="TitlepageSeriesTitleser"] {
+  padding-top: 14pt;
+  margin-top: 11pt;
+  background-image: url("http://www.macmillan.tools.vhost.zerolag.com/bookmaker/bookmakerassets/titlepage-rule.jpg");
+  background-size: 80.4pt auto;
+  background-position: top;
+  background-repeat: no-repeat; }
+
+*[class*="TitlepageTranslatorNametran"],
+*[class*="TitlepageContributorNamecon"] {
+  text-align: center; }
+
+section[data-type="titlepage"] *[class*="TitlepageAuthorNameau"] {
+  font-size: 24pt;
+  line-height: 48pt;
+  margin-top: 108pt;
+  text-indent: 0;
+  text-align: center; }
+
+section[data-type="titlepage"] *[class*="TitlepageImprintLineimp"] {
+  float: bottom;
+  text-align: center;
+  text-indent: 0; }
+
+*[class*="TitlepageCitiescit"], *[class*="TitlepagePublisherNamepub"], *[class*="TitlepageImprintLineimp"] {
+  font-variant: small-caps;
+  font-variant: prince-opentype(smcp, onum);
+  font-style: normal;
+  font-weight: normal;
+  text-transform: lowercase;
+  font-size: 10pt;
+  line-height: 13pt;
+  float: bottom;
+  text-align: center;
+  text-indent: 0; }
+
+
+section[data-type="copyright-page"] p:first-of-type {
+  /*background-image: url("../../images/holt/HoltLogo30K.png");*/
```

Comment:
I want to source it from github, but the url wasn't accurate until this is merged in.  Will go ahead and delete the other two lines..  good catch!